train.py

```diff
@@ -66,7 +66,9 @@ def train_gpu(rank, world_size, opt, dataset, dataset_temporal):
     if not opt.warning_mode:
         warnings.simplefilter("ignore")
 
-    torch.cuda.set_device(opt.gpu_ids[rank])
+    if torch.cuda.is_available():
```

Comment:
Same thing, need an option to allow running joligan on cpu with a gpu machine. An essential use case for this would be cpu tests to ensure we don't break the cpu path in joligan

Reply:
done, `gpu_ids` set to -1.